### PR TITLE
Embed Youtube videos

### PR DESCRIPTION
**Pull request by manuka99**

### CODE REVIEW

**Comment by sithumonline** [comment]
Hi @manuka99, good job...! You missed one other youtube video in hear https://github.com/sliit-foss/zero2hero/blob/master/docs/vscode.md#L5 so can you add it?

File: docs/node/node.md [hunk 1]
```diff
@@ -7,37 +7,39 @@ sidebar_position: 1
...
 ## Install Node and npm
 
 First of all download LST version of node from [here](https://nodejs.org/en/) and install.
 
 ## Install Yarn
 
-`
-npm install -g yarn
-`
+`npm install -g yarn`
```
Comment (sithumonline) [inline docs/node/node.md:29]: can you change this as
```bash
npm install -g yarn
```
Rep : https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting

File: docs/node/node.md [hunk 2]
```diff
@@ -7,37 +7,39 @@ sidebar_position: 1
...
-`
+`npm install -g create-react-app`
 
-## Install Angular CLI 
+## Install Angular CLI
 
-`
-npm install -g @angular/cli
-`
+`npm install -g @angular/cli`
```
Comment (sithumonline) [inline docs/node/node.md:37]:
```bash
npm install -g @angular/cli
```

File: docs/node/node.md [hunk 3]
```diff
@@ -7,37 +7,39 @@ sidebar_position: 1
...
 
 ## How to start development of any JavaScript App?
 
 Clone the repo what will you gonna contribute then run below commands inside the repo
 
 `yarn install`
 
-`yarn start`
\ No newline at end of file
+`yarn start`
```
Comment (sithumonline) [inline docs/node/node.md:45]:
```bash
yarn start
```

**Review by sithumonline — CHANGES_REQUESTED** [review]

File: docs/node/node.md (hunk 1 above)
Comment (manuka99) [reply docs/node/node.md]: That was formatted by the beautify extension, i will fix them

**Comment by manuka99** [comment]
I changed them, can you check them (second commit)

**Pull request merged**